panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp

```diff
@@ -1,567 +1,451 @@
+        dependencies.add("/PanoramaPublic/css/typeahead-examples.css");
+        dependencies.add("/PanoramaPublic/css/PanoramaWebSearch.css");
+    }
+%>
+
+<div class="active-tabs">
+    <input style="visibility: hidden" type="radio" name="active_tabs" id="expSearchPanel" class="search-panel-btn-1" checked >
+    <label for="expSearchPanel" class="search-panel-btn" >Experiment Search</label>
+
+    <input style="visibility: hidden" type="radio" name="active_tabs" id="proteinSearchPanel" class="search-panel-btn-2">
+    <label for="proteinSearchPanel" class="search-panel-btn" >Protein Search</label>
+
+    <input style="visibility: hidden" type="radio" name="active_tabs" id="peptideSearchPanel" class="search-panel-btn-3">
+    <label for="peptideSearchPanel" class="search-panel-btn" >Peptide Search</label>
+
+    <div class="tabs-container">
+        <div class="tab-1">
+            <table class="lk-fields-table">
+                <tr style="height: 10px"></tr>
+                <tr>
+                    <td style="width: 5px"></td>
+                    <td>Author:</td>
+                    <td style="width: 5px"></td>
+                    <td>Title:</td>
+                    <td style="width: 5px"></td>
+                    <td>Organism:</td>
+                    <td style="width: 5px"></td>
+                    <td>Instrument:</td>
+                </tr>
+                <tr>
+                    <td style="width: 5px"></td>
+                    <td nowrap><input class="bootstrap-tagsinput" size="20" type="text" id="Authors" name="Authors" value=""/></td>
+
+                    <td style="width: 5px"></td>
+                    <td><input class="bootstrap-tagsinput" size="20" type="text" id="Title" name="Title" value=""/></td>
+
+                    <td style="width: 5px"></td>
+                    <td nowrap>
+                        <div id="input-picker-div-organism" class="scrollable-dropdown-menu">
+                            <input class="tags" size="20" type="text" id="Organism" name="Organism" placeholder="Enter Organism" value=""/>
+                        </div>
+                    </td>
+
+                    <td style="width: 5px"></td>
+                    <td nowrap>
+                        <div id="input-picker-div-instrument" class="scrollable-dropdown-menu">
+
+                            <%-- Placeholder text is needed here otherwise the input size becomes variable once a value is selected from auto-complete dropdown;
+                            however, hiding the placeholder text by setting 'input::placeholder' styling to be transparent in PanoramaWebSearch.css.
+                            Also, placeholder text is 20 characters long (with added spaces) to keep it consistent with other inputs above.
+                            --%>
+                            <input class="tags" type="text" id="Instrument" name="Instrument" placeholder="  Enter Instrument  " value=""/>
```

Comment:
@labkey-jeckels 
Based on your comment about setting the width instead of invisible placeholder - that is what i had before, but it resets, so resorted to this way

Reply:
OK